runtime-sdk/src/modules/consensus/types.rs

```diff
@@ -0,0 +1,27 @@
+use oasis_core_runtime::common::namespace::Namespace;
+
+/// Kind of root.
+#[derive(Clone, Debug, Default, cbor::Encode, cbor::Decode)]
+#[cfg_attr(test, derive(PartialEq, Eq))]
+#[repr(u8)]
+pub enum RootKind {
+    #[default]
+    Invalid = 0,
```

Comment:
Can we remove this?

Reply:
This follows the same pattern as it is already used by the other internal call in the code (`consensus.TakeReceipt`). Not actually sure, but I guessed we have the invalid as the default variant as an additional safeguard, since the cbor encoding can be a bit messy in soldity. Or my other guess would be it has to do with gas costs in evm.

@kostko please confirm what is the reasoning and if it makes sense to have it here as well.

Reply:
I'm not sure why this would be useful, we could just drop the invalid variant and start the valid ones with 1 (as it is now).

Reply:
Ok I removed the Invalid variant here.